module/data/item/maneuver.mjs

```diff
@@ -1,11 +1,11 @@
-import SystemDataModel from "../abstract.mjs";
 import ItemDescriptionTemplate from "./templates/item-description.mjs";
+import NoneNamegiverPowerData from "./templates/none-namegiver-power.mjs";
 
 /**
  * Data model template with information on Maneuver items.
  * @property {number} extraSuccesses        extra successes to trigger the maneuver
  */
-export default class ManeuverData extends SystemDataModel.mixin(
+export default class ManeuverData extends NoneNamegiverPowerData.mixin(
```

Comment:
Is that true? NPCs can't have maneuvers? KnackManeuvers dont count here?

Reply:
ManeuverData are the maneuver abilities of creatures, like enrage or provoke, they are different to Knack maneuvers. they have the name and the usage of extra successes in common, besides that, the knacks have all the knack stuff on top of it as well. We currently have the differentiation between knackManeuver and Maneuver (may be thats wrong and can be mixxed in later) 
NPCs can have both. maybe the naming of "none-namegiver-powers" is not correct since it is basically for all sentients except characters

Reply:
turned out to be obsolete with update of other branches